plugins/nf-google/src/main/nextflow/cloud/google/batch/GoogleBatchTaskHandler.groovy

```diff
@@ -224,19 +224,8 @@ class GoogleBatchTaskHandler extends TaskHandler implements FusionAwareTask {
             .addAllCommands( cmd )
             .addAllVolumes( launcher.getContainerMounts() )
 
-        final accel = task.config.getAccelerator()
-        // add nvidia specific driver paths
-        // see https://cloud.google.com/batch/docs/create-run-job#create-job-gpu
-        if( accel && accel.type.toLowerCase().startsWith('nvidia-') ) {
-            container
-                .addVolumes('/var/lib/nvidia/lib64:/usr/local/nvidia/lib64')
-                .addVolumes('/var/lib/nvidia/bin:/usr/local/nvidia/bin')
-        }
-
         def containerOptions = task.config.getContainerOptions() ?: ''
-        // accelerator requires privileged option
-        // https://cloud.google.com/batch/docs/create-run-job#create-job-gpu
-        if( task.config.getAccelerator() || fusionEnabled() ) {
+        if( fusionEnabled() ) {
```

Comment:
Why is this logic removed?

Reply:
It does not seem to be needed anymore since Google does this for you automatically if `installGpuDrivers` bit is set. It is not mentioned in the official Google documentation as well.

The only thing is that people might still need to add `/usr/local/nvidia/bin` to their PATH and `/usr/local/nvidia/lib64` to their LD_LIBRARY_PATH depending on their app (some apps automatically look there anyway), but it is not happening right now as well.

Reply:
Indeed, the library path should be handled by the container